### ISSUE TITLE
Bug - Accessibility - Images of Text 

### ISSUE THREAD

**Issue by JamesONeill-DoF** [dof-dss/architecture-blog#29]
General - Multiple images of text appear throughout the site.
Avoid using images of text as screen-reader users cannot access this content. 
Either add alt text to be the same as the text in the image or add this content as text using HTML mark-up.
